test.js

```diff
@@ -447,3 +450,60 @@ test('alias - require previousId and userId', t => {
     })
   })
 })
+
+test.cb('e2e', t => {
```

Comment:
You should make this test `async` and remove `cb` mode. All of the code in it can be converted to `async/await`.

Reply:
Done.